Parser/asdl_c.py

```diff
@@ -1473,6 +1452,7 @@ def write_source(f, mod):
     f.write('#include "Python.h"\n')
     f.write('#include "%s-ast.h"\n' % mod.name)
     f.write('#include "structmember.h"         // PyMemberDef\n')
+    f.write('#include "pycore_pylifecycle.h"   // export _PyAST_Fini()\n')
```

Comment:
This code line occurs `Py_BUILD_CORE` error

Reply:
It seems like `extern void _PyAST_Fini(PyThreadState *tstate);` is not needed to export the symbol. I just removed the include.